project/Dependencies.scala

```diff
@@ -30,13 +30,8 @@ object Dependencies {
         case _                       ⇒ "1.13.2"
       }),
     scalaTestVersion := "3.0.4",
-    java8CompatVersion := {
-      CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, n)) if n >= 13 ⇒ "0.9.0"
-        case Some((2, n)) if n == 12 ⇒ "0.8.0"
-        case _                       ⇒ "0.7.0"
-      }
-    })
+    java8CompatVersion := "0.9.0"
+    )
```

Comment:
I didn't see why this was scala version specific and i wanted to use @johanandren's latest feature :)

Reply:
it's because it's a transitive dependency and not binary compatible between those versions so we can't just drop in a new version in Akka 2.5.x